sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -242,6 +246,58 @@ private static synchronized void init(
     notifyOptionsObservers(options);
 
     finalizePreviousSession(options, HubAdapter.getInstance());
+
+    handleStartupProfilingConfig(options, options.getExecutorService());
+  }
+
+  @SuppressWarnings("FutureReturnValueIgnored")
+  private static void handleStartupProfilingConfig(
+      final @NotNull SentryOptions options,
+      final @NotNull ISentryExecutorService sentryExecutorService) {
+    sentryExecutorService.submit(
```

Comment:
`m`: I think you also need to wrap `submit` into a try-catch, in case e.g. RejectedExecutionException

Reply:
Also addressed in the 3rd PR, will resolve

Reply:
yep, i forgot it, but did it on the third PR